ADVANCED.md

```diff
@@ -182,18 +182,19 @@ ex.
+| `setSearchTerm`     | `searchTerm` String<br/> `options` Object<br/>`options.refresh` Boolean - Refresh search results on update. Default: `true`.<br/>`options.debounce` Number - Length to debounce any resulting queries<br/>`options.autocompleteSuggestions` Boolean - Fetch query suggestions for autocomplete on update, stored in `autocompletedSuggestions` state<br/>`options.autocompleteResults` Boolean - Fetch results on update, stored in `autocompletedResults` state |        |                                                                                                                                           |
+| `setSort`           | `sortField` String - field to sort on<br/>`sortDirection` String - "asc" or "desc"                                                                                                                                                                                                                                                                                                                                                                               |        |                                                                                                                                           |
+| `trackClickThrough` | `documentId` String - The document ID associated with the result that was clicked<br/>`tag` - Array[String] Optional tags which can be used to categorize this click event                                                                                                                                                                                                                                                                                       |        | Report a clickthrough event, which is when a user clicks on a result link.                                                                |
+| `a11yNotify`        | `messageFunc` String - object key to run as function<br/>`messageArgs` Object - Arguments to pass to form your screen reader message string                                                                                                                                                                                                                                                                                                                      |        | Reads out a screen reader accessible notification. See `a11yNotificationMessages` under [Advanced Configuration](#advanced-configuration) |
```

Comment:
Sorry about all the whitespace noise here - definitely looking forward to when we have a new documentation site (hopefully one that is less reliant on markdown tables!)

Reply:
That's ok, these changes are very easy to review if "rich diff" is turned on :)
![image](https://user-images.githubusercontent.com/11838280/61635825-c863b800-ac6a-11e9-8709-1a7492f759a9.png)

Reply:
Oh snap! That's my first time seeing that feature (I've been living in GitHub enterprise the last year which I think is a bit behind in some things), that's super awesome. Thanks for the hot tip @yakhinvadim!

Reply:
The "Hide Whitespace" option can help also.